Automattic/wp-calypso
# Stats: Update Store main chart styling

#### Proposed Changes

<img width="1126" alt="截圖 2022-11-02 下午12 30 18" src="https://user-images.githubusercontent.com/6869813/199397473-b49d2a11-aa59-4b54-a8c4-740295d94b2f.png">

* Adjust styling on the Stats `Store` tab page to align with the new design.
* Try to extract the same styles within the same structure from the `Traffic` and `Store` pages.

#### Testing Instructions



* Spin up a `Jurassic Ninja` site and activate the `WooCommerce` plugin.
* Launch the `local` development application.
* Navigate to the page with the JN site `/store/stats/orders/week/${woo-commerce-site}`.
* Ensure styling on the `Store` tab page is aligned with the new version of the design.
* _**Since the purchase flow would not be easy to simulate, modifying code locally may be easier to test.**_

> Apply some mock data like `Math.random` to the [value and nestedValue](https://github.com/Automattic/wp-calypso/blob/b9bf4d119e8fe789b75008a039c49496d763c02f/client/my-sites/store/app/store-stats/store-stats-chart/index.js#L108-L109) to make bar charts visible.

> Manually replacing [class names to the Delta component](https://github.com/Automattic/wp-calypso/blob/b9bf4d119e8fe789b75008a039c49496d763c02f/client/my-sites/store/app/store-stats/store-stats-orders-chart/index.js#L70) with `is-favorable is-increase` or `is-unfavorable is-decrease` makes the Delta display the new styles.

#### Pre-merge Checklist






Related to #69041
